src/components/Editor/FreeBusy/FreeBusy.vue

```diff
@@ -196,6 +196,9 @@ export default {
 		 */
 		colorCaption() {
 			return [{
+				label: this.$t('calendar', 'Free'),
```

Comment:
Maybe add a `// TRANSLATORS` comment above to indicate how to translate `Free` (as in available).

Reply:
Done